Kickstarter-iOS/ViewModels/AppDelegateViewModel.swift

```diff
@@ -369,6 +372,17 @@ public final class AppDelegateViewModel: AppDelegateViewModelType, AppDelegateVi
       .filter { $0 == .tab(.search) }
       .ignoreValues()
 
+    self.goToLandingPage = self.applicationLaunchOptionsProperty.signal.ignoreValues()
+      .takeWhen(self.optimizelyConfiguredWithResultProperty.signal
+        .skipNil()
+        .filter { $0.isSuccess }
+        .ignoreValues()
+      )
+      .observeForUI()
+      .map {
+        LandingPageViewController()
```

Comment:
This is temporary and will be changed once the experiment and the "first session" logic are implemented (in a different PR).